### PR TITLE
Combine samples into monolithic build

### PR DESCRIPTION
**Pull request by rajdeepnanua-okta**
Moved lots of files around to make monolithic build. All the moves are separated into different commits.
I also had to enable jetifier in gradle.properties to compile totp sample. That will be removed with the new totp sample

### CODE REVIEW

File: browser-sign-in/build.gradle [hunk 1]
```diff
@@ -1,19 +1,73 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
```
Comment (rajdeepnanua-okta) [inline browser-sign-in/build.gradle:1]: This used to be browser-sign-in/app/build.gradle. browser-sign-in/build.gradle is moved to samples-android/build.gradle

**Review by JayNewstrom — APPROVED** [review]
Overall this looks good. I'm mostly trusting you with it, since it's a lot of nitty changes/moves.

File: totp/build.gradle [hunk 2]
```diff
@@ -1,25 +1,34 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
```
Comment (rajdeepnanua-okta) [inline totp/build.gradle:1]: totp/app/build.gradle -> totp/build.gradle

File: LICENSE [hunk 3]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2022 Okta
```
[the remark this answers is not in the document]
Comment (rajdeepnanua-okta) [reply LICENSE]: Good catch! Didn't see the broader context to this before

**Pull request merged**